src/ngraph/frontend/onnx_import/op/batch_norm.cpp

```diff
@@ -47,18 +61,46 @@ namespace ngraph
                     // TODO: Implement learning mode support
                     // float momentum{node.get_attribute_value<float>("momentum", 0.9f)};
                     ASSERT_IS_SUPPORTED(node, is_test) << "only 'is_test' mode is supported.";
-                    ASSERT_IS_SUPPORTED(node, spatial) << "only 'spatial' mode is supported.";
 
-                    if (inputs.size() >= 5)
+                    if (spatial == 0)
                     {
-                        mean = inputs.at(3);
-                        var = inputs.at(4);
-                        return {std::make_shared<ngraph::op::BatchNormInference>(
-                            epsilon, scale, bias, x, mean, var)};
+                        auto mean =
+                            legacy_style_broadcast_for_binary_operation(data, inputs.at(3), 1)
+                                .at(1);
+                        auto variance =
+                            legacy_style_broadcast_for_binary_operation(data, inputs.at(4), 1)
+                                .at(1);
+
+                        bias = legacy_style_broadcast_for_binary_operation(data, bias, 1).at(1);
```

Comment:
Do we need to mix `legacy_style_broadcast_for_binary_operation` and `numpy_style_broadcast_for_binary_operation` in this implementation? Would it be possible to stick to just one of them?